exercises/practice/parallel-letter-frequency/.docs/instructions.append.md

```diff
@@ -0,0 +1,9 @@
+# Instructions append
+
+## Using Parallelism
+
+The goal of this exercise is to practice parallelism with Emacs Lisp.
+
+In Emacs Lisp this is achived by using [`asynchronous processes`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Asynchronous-Processes.html#:~:text=An%20asynchronous%20process%20is%20controlled,%2Dtype%20(see%20below)).
+
+You may also want to look at the documentation for [`batch mode`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Batch-Mode.html), [`sentinels`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Sentinels.html) and [`receiving output from processes`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Output-from-Processes.html).
```

Comment:
My only worry is that this is too much information / might be better included as a hint.

From the docs:

> The hints should not spell out the solution, but instead point to a resource describing the solution (e.g. linking to documentation for the function to use).

- https://exercism.org/docs/building/tracks/practice-exercises#h-file-docs-hints-md

@kmarker1101 Do you think this is information that is needed to get on the right track or is this info that the student should only see once they're frustrated and want some additional guidance?

Reply:
@fapdash I think it is probably ok as is. I think if we don't supply the information upfront, students will google and find async.el, which would require adding the async.el package. Not sure we want to do that?

Thanks for adding support for the extra file. Yes, it was painful writing inside the string. LOL